src/python/pants/reporting/reporter.py

```diff
@@ -10,6 +10,11 @@
 from pants.reporting.report import Report
 
 
+class ReporterDestination(object):
+  OUT = 'out'
+  ERR = 'err'
+
```

Comment:
why not `int` constants vs strings?

Reply:
Was thinking debuggability, but I guess there would actually be a slight performance difference here, huh? Will switch.